### ISSUE TITLE
I successfully ran this process, but I found some issues

### ISSUE THREAD

**Issue by zxb-0** [arthurfenderbucker/LaTTe-Language-Trajectory-TransformEr#4]
Thank you very much for your excellent work, but I have a question for you, and I would appreciate it if you could take some time out of your busy schedule to answer me.The model I use is the model of the training number in the paper. GPU: Two RTX 2080Ti, the second half of the model that ran out of the trajectory is obviously wrong.
![VR@JG{X8D_X9844KTZ5M_JI](https://github.com/arthurfenderbucker/LaTTe-Language-Trajectory-TransformEr/assets/97783815/b3689892-da4d-4de1-a224-163b5a3dea9e)
The graph below shows my GPU usage
![D~N` QB7F}ACV(FK11OS(EE](https://github.com/arthurfenderbucker/LaTTe-Language-Trajectory-TransformEr/assets/97783815/c5387c4f-7c74-4fd9-b922-9197dabca970)
Is it because of a problem with my GPU, or is there a problem with the model I downloaded?

**Comment by arthurfenderbucker** [comment]
Hello @zxb-0, I appreciate the interest for our work. The GPU usage looks indeed much higher than normal. Are you running the experiments in a Jupyter notebook? Sometimes loading the model several times in the same Jupyter kernel could lead to this excessive use of GPU memory.
Regarding the plot, is this a recurring error or just in this specific trajectory? If it is happening with all the environments, I would infer that the system was not setup right. However, in the case of isolated error, I would assume that it is an expected behavior. As reported in the paper, the model has shown faulty behaviors in a small portion of the environments tested.

**Comment by zxb-0** [comment]
@arthurfenderbucker  This is a recurring mistake. I'm running a interactive.py and I'm not using a Jupyter notebook, and then this is the track, and no matter how much I change the text input, the second half of the track will end up with obvious errors.

**Comment by zxb-0** [comment]
@arthurfenderbucker I'm pretty sure I just ran interactive.py and the GPU usage was always that high. When I run model_overview_4D.ipynb, the GPU usage is just as high, and there are also errors. This is shown in the figure below. All the configurations of the project are from the readme. I kindly ask you to help me take a look, thank you very much.
![L 8RCM KR0FMAM6((6 11L7](https://github.com/arthurfenderbucker/LaTTe-Language-Trajectory-TransformEr/assets/97783815/ee1d8d79-f627-4d42-9e40-01775a0a784c)

**Comment by zxb-0** [comment]
@arthurfenderbucker Can you send me a copy of your trained model, thank you very much

**Comment by arthurfenderbucker** [comment]
Hello @zxb-0, I just tested the pipeline in a new machine. The trained model and dataset are the ones provided in the README and they seem to be working fine.

I apologize for the confusing repo structure. To mitigate that, I just created a new cleaner notebook to run a simple interactive demo using the model and dataset given. Please check if you can run the demo [interactive_user_study.ipynb](https://github.com/arthurfenderbucker/LaTTe-Language-Trajectory-TransformEr/blob/main/interactive_user_study.ipynb) 

Relevant notes for the setup:
- Make sure that the src/config.py file is points to the right directories of the models and data.
- I got a GPU usage of 7822MiB in total (tested on a RTX4070 8G). I don't know if it would change in a system with more than one GPU, but I suppose not.
- It is also possible to run it on CPU only by adding ```os.environ['CUDA_VISIBLE_DEVICES'] = '-1' ```  before loading the modules

**Comment by zxb-0** [comment]
@arthurfenderbucker Thank you very much for your help, I can run interactive_user_study.ipynb successfully. Regarding the issue of high GPU usage, I have found something here. If I am in【 model_overview_4D.ipynb】, when the program reaches the 【reset_seed(seed = seed) 】sentence of In[1], the GPU usage is suddenly close to full. More precisely, when running into the 【tf.random.set_global_generator(tf.random.Generator.from_seed(seed)) 】statement in the reset_seed method. The GPU usage is suddenly full.The same problem occurs when running on other computers similarly. when the program is running to 【X_train, X_test, X_valid, y_train, y_test, y_valid, indices_train, indices_test, indices_val = mr.split_dataset(X, Y, test_size=0.2, val_size=0.1)】sentence of In[4] in【 model_overview_4D.ipynb】, the GPU occupancy rate is suddenly close to full.
moreover In your【 spec-file .txt】, you used the 【https://conda.anaconda.org/pytorch/noarch/torchvision-0.2.2-py_3.tar.bz2 】in the previous version, but I didn't find this package, so I 【https://conda.anaconda.org/pytorch/noarch/torchvision-cpu-0.2.2-py_3.tar.bz2】 this package instead.In your latest 【spec-file.txt】, you commented two packages, including the package mentioned above, but after I downloaded your latest process and successfully loaded it, I found that if there is no package, an error will be reported. I'm guessing that this bag should be important, so is it the reason I changed this bag.Can you update your spec-file.txt,I would like to ask if you can check if the project you uploaded is the same as the one you have on your computer I would appreciate it.I wish you all the best.

![~JDE6PY3CM1WASNP C3U4$X](https://github.com/arthurfenderbucker/LaTTe-Language-Trajectory-TransformEr/assets/97783815/f846bd2b-d5af-4b5d-96bd-6e5a7599823a)
